apps/builddao/widget/components/profile/AccountFeed.jsx

```diff
@@ -0,0 +1,198 @@
+const { Post } = VM.require("buildhub.near/widget/components") || (() => <></>);
+
+const indexKey = props.indexKey ?? "main";
+const groupId = props.groupId;
+const permissions = props.permissions;
+
+const index = [
+  {
+    action: "post",
+    key: indexKey,
+    options: {
+      limit: 10,
+      order: "desc",
+      accountId: props.accounts,
+    },
+    cacheOptions: {
+      ignoreCache: true,
+    },
+  },
+  {
+    action: "repost",
+    key: indexKey,
+    options: {
+      limit: 10,
+      order: "desc",
+      accountId: props.accounts,
+    },
+    cacheOptions: {
+      ignoreCache: true,
+    },
+  },
+];
+
+const isPremiumFeed = props.isPremiumFeed;
+const commentAccounts = props.commentAccounts;
+const renderedPosts = {};
+
+const makePostItem = (a) => ({
+  type: "social",
+  path: `${a.accountId}/post/main`,
+  blockHeight: a.blockHeight,
+});
+
+const renderPost = (a) => {
```

Comment:
This all could be replaced with devs.near/widget/Feed module if you want, just set index to post/main, repost/main

Reply:
I think I'll leave it as is...